### PR TITLE
Fix for handling incomplete file

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes are proposed in this pull request?

- Change the verbose warn log for incomplete file to debug level.
- Remove wait for complete file from getattr.
- Reduce wait time for open

### CODE REVIEW

**Comment by jiacheliu3** [comment]
@maobaolong When do you typically see this incomplete file warning? Could you explain the use case a bit? From my understanding, if a batch job getStatus before the write is finished, the job can probably be improved to do getStatus later. But seems from your case you are using FUSE, right?

**Comment by maobaolong** [comment]
@jiacheliu3 Yeah, we use FUSE to access Alluxio with ufs Ozone, the workload is fio test.

**Comment by LuQQiu** [comment]
@maobaolong can you help fix the merge conflicts

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java [hunk 1]
```diff
@@ -549,7 +553,7 @@ private int openInternal(String path, FuseFileInfo fi) {
       try {
         is = mFileSystem.openFile(uri);
       } catch (FileIncompleteException e) {
-        if (AlluxioFuseUtils.waitForFileCompleted(mFileSystem, uri)) {
+        if (AlluxioFuseUtils.waitForFileCompleted(mFileSystem, uri, 1000)) {
```
Comment (LuQQiu) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java:556]: why choosing a shorter duration?

**Review by LuQQiu — COMMENTED** [review]
Sorry that many changes go in to Fuse recently and it may be easier to recreate the PRs with those changes

**Comment by maobaolong** [comment]
@LuQQiu Please take another look, thank you.

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java [hunk 2]
```diff
@@ -264,22 +264,12 @@ private int getattrInternal(String path, FileStat stat) {
...
       if (!status.isCompleted()) {
-        if (mCreateFileEntries.contains(PATH_INDEX, path)) {
-          // Alluxio master will not update file length until file is completed
-          // get file length from the current output stream
-          CreateFileEntry<FileOutStream> ce = mCreateFileEntries.getFirstByField(PATH_INDEX, path);
-          if (ce != null) {
-            FileOutStream os = ce.getOut();
-            size = os.getBytesWritten();
-          }
-        } else if (!AlluxioFuseUtils.waitForFileCompleted(mFileSystem, uri)) {
```
Comment (LuQQiu) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:275]: @maobaolong 
Removing the wait will introduce problems in the following workload
1. Fuse write - then getattr() - then possibly read() workload.
Fuse.write() - Fuse.release() which (1)  mCreateFileEntries.remove(ce); (2)  ce.close();
then the new Fuse.getattr() without waiting for file completed will return size = 0, cause the application thinking that the file write failed and error out.
The wait is added to workaround the async release().

**Comment by LuQQiu** [comment]
Integration tests also failed
```
Error: 1.451 [ERROR] Failures: 
Error: 1.452 [ERROR] alluxio.client.fuse.JNIFuseIntegrationTest.ls
Error: 1.454 [ERROR]   Run 1: JNIFuseIntegrationTest>AbstractFuseIntegrationTest.ls:249 expected:<[40K]> but was:<[0]>
Error: 1.456 [ERROR]   Run 2: JNIFuseIntegrationTest>AbstractFuseIntegrationTest.ls:249 expected:<[40K]> but was:<[0]>
15:38:11.457 [INFO] 
Error: 1.457 [ERROR] alluxio.client.fuse.JNRFuseIntegrationTest.ls
Error: 1.457 [ERROR]   Run 1: JNRFuseIntegrationTest>AbstractFuseIntegrationTest.ls:249 expected:<[40K]> but was:<[0]>
Error: 1.457 [ERROR]   Run 2: JNRFuseIntegrationTest>AbstractFuseIntegrationTest.ls:249 expected:<[40K]> but was:<[0]>
```

**Comment by maobaolong** [comment]
@LuQQiu Revert all changes except the log warn

**Review by LuQQiu — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**